assignments/assignment01.rb

```diff
@@ -38,7 +38,8 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
-  # your implementation here
+  string = s.split(" ").map(&:capitalize).join(" ") 
+  puts string
```

Comment:
You want to return the string, you don't want to print it out. So, this line shouldn't be here.